test/polygon/sand/fixtures.ts

```diff
@@ -6,16 +6,42 @@ import {
 } from 'hardhat';
 
 import {setupUsers, setupUser} from '../../utils';
+import {Contract} from 'ethers';
 
+type User = {address: string; Sand: Contract; TrustedForwarder: Contract};
+export type Fixtures = {
+  Sand: Contract;
+  users: User[];
+  sandBeneficiary: User;
+  deployer: User;
+  ChildChainManager: Contract;
+  TrustedForwarder: Contract;
+};
 export const setupSand = deployments.createFixture(async () => {
   await deployments.fixture('PolygonSand');
   const Sand = await ethers.getContract('PolygonSand');
   const accounts = await getNamedAccounts();
   const unnamedAccounts = await getUnnamedAccounts();
-  const users = await setupUsers(unnamedAccounts, {Sand});
-  const childChainManager = await ethers.getContract('CHILD_CHAIN_MANAGER');
+  const ChildChainManager = await ethers.getContract('CHILD_CHAIN_MANAGER');
```

Comment:
variables should not start with an upper case letter, not sure why you changed this (i know sand has an upper case)

Reply:
yes I tried to mimic what was there, and also is pretty common to user upper case for contracts (on the js side).
I will change back everything to lower case.